.github/workflows/ci.yml

```diff
@@ -172,7 +174,7 @@ jobs:
           extra_test_args: --workspace --exclude tokio-boring --exclude hyper-boring
         - thing: i686-msvc
           target: i686-pc-windows-msvc
-          rust: stable-x86_64-msvc
+          rust: stable-i686-msvc
```

Comment:
I should probably ask, but I'm not sure if I want to 🙈

Reply:
This was from the previous Cloudflare PR, but I just reverted it all and constrained the change to just FIPS.